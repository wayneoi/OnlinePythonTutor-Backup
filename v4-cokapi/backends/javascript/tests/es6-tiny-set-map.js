var mySet = new Set();
var myMap = new Map();
