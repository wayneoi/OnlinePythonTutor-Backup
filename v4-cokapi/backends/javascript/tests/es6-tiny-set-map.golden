{ code: 'var mySet = new Set();\nvar myMap = new Map();',
  trace: 
   [ { stdout: '',
       func_name: '',
       stack_to_render: [],
       globals: 
        { mySet: [ 'JS_SPECIAL_VAL', 'undefined' ],
          myMap: [ 'JS_SPECIAL_VAL', 'undefined' ] },
       ordered_globals: [ 'mySet', 'myMap' ],
       line: 1,
       col: 0,
       event: 'step_line',
       heap: {} },
     { stdout: '',
       func_name: '',
       stack_to_render: [],
       globals: 
        { mySet: [ 'REF', 1 ],
          myMap: [ 'JS_SPECIAL_VAL', 'undefined' ] },
       ordered_globals: [ 'mySet', 'myMap' ],
       line: 2,
       col: 0,
       event: 'step_line',
       heap: { '1': [ 'SET' ] } },
     { stdout: '',
       func_name: '',
       stack_to_render: [],
       globals: { mySet: [ 'REF', 1 ], myMap: [ 'REF', 2 ] },
       ordered_globals: [ 'mySet', 'myMap' ],
       line: 2,
       col: 22,
       event: 'step_line',
       heap: { '1': [ 'SET' ], '2': [ 'DICT' ] } } ] }
